### DISCUSSION TITLE
📚 iCub Tech Docs

### DISCUSSION THREAD

**Discussion by pattacini** [robotology/community#405] [Announcements]
Hey @robotology/everyone 👋 

We'd like to let you know that we have just published the [**`iCub Tech Docs`**][1], which is by now a handful of instructions that you may find helpful when fiddling with the iCub tendons, especially when you need to replace broken parts.

Further, the same website provides a list of mechanical upgrade kits you can request for your iCub.

These new resources do complement our official documentation and are meant to be ever-growing so that they will likely encompass other important topics in the near future.

[Contributions][2] in terms of fixes and new additions are very welcome and very easy to do through [forks and pull-requests][3].

Stay tuned 🎧 

[1]: https://github.com/icub-tech-iit/documentation
[2]: https://github.com/icub-tech-iit/documentation/blob/master/.github/CONTRIBUTING.md
[3]: https://guides.github.com/activities/forking

**Comment by DanielePucci** [comment]
CC @robotology/iit-dynamic-interaction-control
